src/NerdWiki.js

```diff
@@ -0,0 +1,142 @@
+const NerdWiki = (function () {
+    const htmlHelper = (function () {
```

Comment:
is 'htmlHelper' an already existing function in js or did we create it for this specific project?

Reply:
**Explanation of the Code**
What is `const htmlHelper = ...?`

htmlHelper is a variable that we're creating in this project. It is not something that already exists in JavaScript.
In JavaScript, **const** means this variable's value cannot be changed after it's assigned.
What is **(function() { ... })?**

This part is called an Immediately Invoked Function Expression (IIFE).
It's a function that is written and immediately executed (or "invoked") when the program runs.
The purpose of wrapping code in an IIFE is often to create a "private" scope for the code, keeping things organized and avoiding conflicts with other parts of your program.
What does this code do?

We're creating a helper tool called htmlHelper. Inside the function (function() { ... }), we can define methods (like small tools) to help us work with HTML, such as adding, removing, or manipulating elements on a webpage.
Answering the Question
The question asks: Is htmlHelper an already existing function in JS or did we create it for this specific project?

The answer is: We created htmlHelper for this specific project. It is not a built-in JavaScript function or object. This is a custom tool that the developer designed to make working with HTML easier in this project.

Why we created **HtmlHelper**?
I added it because this helps us to organise and maintain our source code better. The htmlHelper contains all the functions that are related with **DOM**  manipulations like:

1. creating elements
2. creating section elements
3. creating section items elements 

all the DOM manipulations are there

It's inside **NerdWiki** because this helps us to encapsulate htmlHelper object. We dont need it to be publically accessible outside the NerdWiki